src/constants/routing.ts

```diff
@@ -72,66 +59,6 @@ const WRAPPED_NATIVE_CURRENCIES_ONLY: ChainTokenList = Object.fromEntries(
     .filter(Boolean)
 )
 
-// used to construct intermediary pairs for trading
-export const BASES_TO_CHECK_TRADES_AGAINST: ChainTokenList = {
-  ...WRAPPED_NATIVE_CURRENCIES_ONLY,
-  [ChainId.MAINNET]: [...WRAPPED_NATIVE_CURRENCIES_ONLY[ChainId.MAINNET], DAI, USDC_MAINNET, USDT, WBTC],
-  [ChainId.OPTIMISM]: [...WRAPPED_NATIVE_CURRENCIES_ONLY[ChainId.OPTIMISM], DAI_OPTIMISM, USDT_OPTIMISM, WBTC_OPTIMISM],
-  [ChainId.ARBITRUM_ONE]: [
-    ...WRAPPED_NATIVE_CURRENCIES_ONLY[ChainId.ARBITRUM_ONE],
-    DAI_ARBITRUM_ONE,
-    USDT_ARBITRUM_ONE,
-    WBTC_ARBITRUM_ONE,
-  ],
-  [ChainId.POLYGON]: [
-    ...WRAPPED_NATIVE_CURRENCIES_ONLY[ChainId.POLYGON],
-    DAI_POLYGON,
-    USDC_POLYGON,
-    USDT_POLYGON,
-    WETH_POLYGON,
-  ],
-  [ChainId.BNB]: [
-    ...WRAPPED_NATIVE_CURRENCIES_ONLY[ChainId.BNB],
-    DAI_BSC,
-    USDC_BSC,
-    USDT_BSC,
```

Comment:
so these were not being used post-uniswapx merge?

Reply:
I was looking through the smart-order-router package for these and I don't see them. I'm a bit concerned that the quality of client-side generated routes has declined.

It's fine to remove this code in this PR since it's dead rn, but I think we should investigate the internals on smart-order-router and ensure this is present, or that it can be surfaced to consumers as a config.